Parser/ExprGrapher.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace PSI;
+class ExprGrapher : Visitor<StringBuilder> {
+   StringBuilder mSB = new ();
+   int id = 1;

```

Comment:
```suggestion
   int mID = 1;

```
Member variable should start with m prefix.